fe/fe-core/src/main/java/com/starrocks/common/util/NetUtils.java

```diff
@@ -60,4 +60,25 @@ public static boolean isPortUsing(String host, int port) throws UnknownHostExcep
         }
         return flag;
     }
+
+    public static boolean validIPAddress(String hostname) {
```

Comment:
There are so many libraries to check the IPAddress in java.
There is no necessarity to write it again.
```
https://commons.apache.org/proper/commons-validator/apidocs/org/apache/commons/validator/routines/InetAddressValidator.html
```